datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/resolvers/role/ListRolesResolver.java

```diff
@@ -41,10 +39,6 @@ public class ListRolesResolver implements DataFetcher<CompletableFuture<ListRole
   @Override
   public CompletableFuture<ListRolesResult> get(final DataFetchingEnvironment environment) throws Exception {
     final QueryContext context = environment.getContext();
-    if (!canManagePolicies(context)) {
-      throw new AuthorizationException(
-          "Unauthorized to view roles. Please contact your DataHub administrator if this needs corrective action.");
-    }
```

Comment:
can everyone view roles? I don't see why we would need to gatekeep that as intensely, just curious.

Reply:
I kinda agree. Why should we guard this one

Reply:
Right, which is why we removed this?